index.css

```diff
@@ -118,4 +121,42 @@ input[type="submit"] {
     border: 2px solid black;
     border-radius: 5px;
     padding: 10px 15px;
+}
+
+
+/* calander
+*/
+.calander {
+    font-size: 24px;
+    font-weight: 600;
+}
+
+.month {
+   text-align: center; 
+}
+
+.weekdays {
+  margin: 0;
+  padding: 10px 0;
+}
+
+.weekdays li {
+  width: calc(97% / 7);
```

Comment:
this here really confuses me cause when i do 100/7 it does not split properly. if anyone knows css pls fix